src/cli_plugin/install/index.js

```diff
@@ -18,6 +19,25 @@ function processCommand(command, options) {
   install(settings, logger);
 }
 
+function getDefaultConfigPath() {
+  const paths = [
+    fromRoot('config/kibana.yml'),
+    '/etc/kibana/kibana.yml'
+  ];
+
+  let defaultPath = paths[0];
```

Comment:
I was on the fence about this - the current behavior is to pass config/kibana.yml whether it exists or not and throw if the file doesn't exist.  This preserves that behavior.  An alternative I was considering was to log a warning and optimize with default settings.

This still does the file check because if config/kibana.yml doesn't exist, we want to try the other path and use that as the default if it exists.